react/week2/todo/src/App.js

```diff
@@ -1,24 +1,24 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from "react";
 import "./App.css";
 import data from "./data/data.json";
 import Header from "./components/Header";
 import Timer from "./components/Timer";
 import TodoList from "./components/TodoList";
 
 function App() {
-  // hooks
-  const [state, setState] = useState(null);
-  if (state === null) setTimeout(() => setState(data), 3000);
-
+  const [time, setTime] = useState(null);
+  useEffect(() => {
+    if (time === null) setTimeout(() => setTime(data), 3000);
+    return () => {
+      clearInterval();
+    };
```

Comment:
I guess you wanted to write `clearTimeout()` here. Nevertheless, it is not necessary to do cleanup for a timeout, since it's called only once (whereas intervals will run infinitely if they're not cleared). So in this case you can omit the entire return statement.